src/pybind/mgr/orchestrator/module.py

```diff
@@ -684,43 +690,31 @@ def _daemon_add_misc(self, daemon_type=None, placement=None, inbuf=None):
         'name=fs_name,type=CephString '
         'name=placement,type=CephString,req=false',
         'Start MDS daemon(s)')
-    def _mds_add(self, fs_name, placement=None):
+    def _mds_add(self, fs_name, placement=None, inbuf=None):
```

Comment:
The difference between:

`ceph orch daemon add mds`

and

`ceph orch daemon add <daemon_type>`

is that the the former accepts a `inbuf` or a `placement` while the latter allows to pass a `fs_name` and a `placement`. As we discussed yesterday, I find this highly confusing.

It's probably out of scope for this PR but I wanted to raise awareness for this issue again.